airbyte-webapp/src/packages/cloud/views/layout/SideBar/SideBar.tsx

```diff
@@ -14,162 +15,150 @@ import { CloudRoutes } from "packages/cloud/cloudRoutes";
 import OnboardingIcon from "views/layout/SideBar/components/OnboardingIcon";
 import SettingsIcon from "views/layout/SideBar/components/SettingsIcon";
-import SidebarPopout, { Icon, Item } from "views/layout/SideBar/components/SidebarPopout";
 import SourceIcon from "views/layout/SideBar/components/SourceIcon";
 import { NotificationIndicator } from "views/layout/SideBar/NotificationIndicator";
-import { useCalculateSidebarStyles, getPopoutStyles } from "views/layout/SideBar/SideBar";
+import { useCalculateSidebarStyles } from "views/layout/SideBar/SideBar";
 
+import { useConfig } from "../../../../../config";
 import { RoutePaths } from "../../../../../pages/routePaths";
-
-const Bar = styled.nav`
-  width: 100px;
-  min-width: 65px;
-  height: 100%;
-  background: ${({ theme }) => theme.darkPrimaryColor};
-  padding: 23px 3px 15px 4px;
-  text-align: center;
-  display: flex;
-  flex-direction: column;
-  justify-content: space-between;
-  position: relative;
-  z-index: 9999;
-`;
-
-const Menu = styled.ul`
-  padding: 0;
-  margin: 20px 0 0;
-  width: 100%;
-`;
-
-const Text = styled.div`
-  margin-top: 7px;
-`;
-
-const WorkspaceButton = styled.div`
-  font-size: 9px;
-  line-height: 21px;
-  font-weight: 400;
-  height: 21px;
-  color: ${({ theme }) => theme.whiteColor};
-  border-radius: 10px;
-  margin-top: 13px;
-  background: rgba(255, 255, 255, 0.2);
-  cursor: pointer;
-  display: block;
-  white-space: nowrap;
-  overflow: hidden;
-  text-overflow: ellipsis;
-  padding: 0 8px;
-  text-align: center;
-`;
+import ChatIcon from "../../../../../views/layout/SideBar/components/ChatIcon";
+import DocsIcon from "../../../../../views/layout/SideBar/components/DocsIcon";
+import RecipesIcon from "../../../../../views/layout/SideBar/components/RecipesIcon";
+import {
+  SidebarDropdownMenu,
+  SidebarDropdownMenuItemType,
+} from "../../../../../views/layout/SideBar/components/SidebarDropdownMenu";
+import StatusIcon from "../../../../../views/layout/SideBar/components/StatusIcon";
+import styles from "./SideBar.module.scss";
 
 const SideBar: React.FC = () => {
+  const navLinkClassName = useCalculateSidebarStyles();
   const workspace = useCurrentWorkspace();
   const cloudWorkspace = useGetCloudWorkspace(workspace.workspaceId);
+  const config = useConfig();
   const { show } = useIntercom();
   const handleChatUs = () => show();
 
-  const navLinkClassName = useCalculateSidebarStyles();
-
   return (
-    <Bar>
+    <nav className={styles.nav}>
       <div>
         <Link to={workspace.displaySetupWizard ? RoutePaths.Onboarding : RoutePaths.Connections}>
           <img src="/simpleLogo.svg" alt="logo" height={33} width={33} />
         </Link>
         <WorkspacePopout>
-          {({ onOpen, value }) => <WorkspaceButton onClick={onOpen}>{value}</WorkspaceButton>}
+          {({ onOpen, value }) => (
+            <button className={styles.workspaceButton} onClick={onOpen}>
+              {value}
+            </button>
```

Comment:
One of the ways do not to handle text styles manually
```suggestion
            <button className={styles.workspaceButton} onClick={onOpen}>
             <Text> {value} </Text>
            </button>
```

Reply:
Fixed! Thanks 😉